general/solomiia_tymoshchuk/lecture3/_2/BirthDayValidator.java

```diff
@@ -0,0 +1,24 @@
+package general.solomiia_tymoshchuk.lecture3._2;
+
+import static java.lang.Integer.parseInt;
+
+public class BirthDayValidator extends AccountValidator
+{
+
+    public BirthDayValidator(Long birthDay){
+        this.birthDay = birthDay;
+    }
+
+    @Override
+    public boolean isValid(){
+        boolean value;
+        int birthDayConverted = parseInt(String.valueOf(birthDay));
```

Comment:
you don't need to convert a value to string and parse it back - you can simply use it directly, since it's already a number